Tag shapes

matthiasblum opened the pull request:
This PR adds two new shapes for `nightingale-track`: `tag` and `doubleEndedTag`.

dlrice commented on packages/nightingale-track/src/FeatureShape.ts:
```diff
@@ -304,4 +309,35 @@ export default class FeatureShape {
+  tag() {
+    const h = this.#ftHeight;
+    const w = this.#ftWidth;
+    const d1 = Math.sqrt(2 * Math.pow(h, 2));
```
```suggestion
    const d1 = Math.sqrt(2) * h
```
This seems equivalent, unless we're guarding specifically against `h` < 0?

dlrice reviewed (commented):
Would you mind updating `/packages/nightingale-track/tests/mockData/data.json` so the story shows these new shapes too?

matthiasblum commented:
- rewrote the method to generate the `tag` shape to support three shapes: `leftEndedTag`, `rightEndedTag`, `doubleEndedTag`
- mock data updated
- fixed typos in the README
